extensions/hadoop/src/test/java/com/hazelcast/jet/hadoop/file/CsvFileFormatTest.java

```diff
@@ -33,7 +33,6 @@
 
     @Test
     public void shouldReadCsvFile() {
-
```

Comment:
We could use a test, proving that _projection_ works.

Reply:
Added a unit test for the `Util.createFieldProjection()` method.

Reply:
I agree there should be some tests in the file connector for this. It is user facing API and there is non trivial amount of new code here. I pushed some.